### ISSUE TITLE
Extension doesn't work when 3rd party cookies are disabled in Chrome

### ISSUE THREAD

**Issue by arendjr** [facebook/react-devtools#84]
When 3rd party cookies are disabled in Chrome, the entire extension fails to load. The React tab shows up next to the Console tab, but when clicked upon shows nothing but an empty white page.

First of all this took me quite a while before I figured out what the problem was. If you're unwilling to fix this bug, at the very least it would be nice to inform people what setting they should toggle in order to get things to work.

Preferably I would like to see the extension work without having to enable 3rd party cookies though :)

Thanks!

**Comment by spicyj** [comment]
This is already noted in the README:

https://github.com/facebook/react-devtools#the-react-tab-is-blank

**Comment by arendjr** [comment]
Aii... my bad for not reading that...

**Comment by spicyj** [comment]
No worries. Where did you look? Maybe we can add a note there too.

**Comment by arendjr** [comment]
Of course I bumped into the empty tab at first. The ideal place for a notice would be there :)

Then I started Googling for a solution and started reading through the reviews on the Chrome Store Page (https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) where eventually I found one that mentioned this issue. But by then I had also found some older questions about the extension not working on Stack Overflow, so I had attempted to use the `window.React = require('react');` hack as well even though the newer version shouldn't require that anymore (and it works without it now).
